openlibrary/templates/site/body.html

```diff
@@ -6,8 +6,11 @@
 $ bodyattrs = ctx.get('bodyattrs', [])
 
 $if ctx.path.startswith('/works/OL') or ctx.path.startswith('/authors/OL') or ctx.path.startswith('/books/OL') or ctx.path.startswith('/search'):
-  $if ctx.user and ((ctx.user.is_librarian() or ctx.user.is_admin()) and ctx.user.is_usergroup_member('/usergroup/librarian-work-merge')):
+  $if ctx.user and ((ctx.user.is_librarian() or ctx.user.is_admin())):
     $bodyclass.append('show-librarian-tools')
+    $bodyattrs.append('data-username="%s"' % ctx.user.key.split('/')[-1])
+    $if ctx.user.is_usergroup_member('/usergroup/librarian-work-merge'):
+      $bodyattrs.append('data-can-merge="%s"' % 'true')
```

Comment:
This assumes that super librarians (members of `/usergroup/librarian-work-merge`) will also be a member of either `/usergroup/admin` or `/usergroup/librarians`.  If not, the super librarian will not have "Merge" links rendered in the table.

I'm also wondering if `admin`s should also be permitted to use the merge UI.  Currently, they can only submit merge requests.